src/app/campaign/store/models/flight-days.models.ts

```diff
@@ -0,0 +1,57 @@
+import { utc } from 'moment';
+import { HalDoc } from 'ngx-prx-styleguide';
+
+export interface FlightPreviewParams {
```

Comment:
(It should be enough to just use the `Flight` for this - one less thing to keep in-sync).